k8s/helm/charts/centrifugo/templates/externalsecret.yaml

```diff
@@ -17,4 +17,4 @@ spec:
   data:
     - secretKey: config.json
       remoteRef:
-        key: {{ $.Release.Namespace }}-centrifugo-config
+        key: {{ .Release.Namespace }}-centrifugo-config
```

Comment:
@AliakseiMalyshau  Is that a mistake?

Reply:
@LiorAgnin not exactly, just typo error. The right was is specify Helm variable like this. Previous variant is valid for the loop.